containers/deploy/foreman/deployments.yml

```diff
@@ -204,3 +213,89 @@
     - start
     - restart
     - destroy
+
+- name: Create deployment, and scale replicas up
+  openshift_v1_deployment_config:
+      state: "{{ deployment_state }}"
+      force: false
+      resource_definition:
+          apiVersion: v1
+          kind: deployment_config
+          metadata:
+              name: httpd
+              namespace: "{{ project_name }}"
+              labels:
+                  app: foreman
+                  service: httpd
+          spec:
+            selector:
+                name: httpd
+            strategy:
+                type: Recreate
+                recreateParams:
+                    timeoutSeconds: 1200
+            triggers:
+                - type: ConfigChange
+            replicas: 1
+            template:
+                metadata:
+                    name: "httpd"
+                    labels:
+                        name: "httpd"
+                spec:
+                    serviceAccount: anyuid
+                    serviceAccountName: anyuid
+                    volumes:
+                      - name: httpd-config
+                        configMap:
+                            name: "httpd-configs"
+                      - name: certs
+                        secret:
+                            secretName: certs
+                            items:
+                              - path: katello-default-ca.crt
+                                key: ca.crt
+                              - path: katello-server.crt
+                                key: pulp.crt
+                      - name: keys
+                        secret:
+                            secretName: keys
+                            items:
+                              - path: katello-default-ca.key
+                                key: ca.key
+                              - path: katello-server.key
+                                key: pulp.key
+                    containers:
+                      - name: httpd
+                        image: "{{ registry }}/foreman-httpd:latest"
+                        command:
+                          - /usr/sbin/init
```

Comment:
I don't think you need to specify this here. This should be in the image.

Reply:
I thought so too, but it was like because it was a derivative image built with a-c that this somehow disappeared and I had to add it to get things running.

Reply:
That's odd ... find with me then. Not something I really want to spend time on at the moment.